flow/project.py

```diff
@@ -263,10 +273,13 @@ class _JobOperation(object):
         :class:`dict`
     """
 
-    def __init__(self, id, name, job, cmd, directives=None):
+    def __init__(self, id, name, jobs, cmd, directives=None):
         self._id = id
         self.name = name
-        self.job = job
+
+        assert isinstance(jobs, tuple)
```

Comment:
This is for code security. If the team want me to remove it, I'll do that.

Reply:
Since this is an internal class, I believe it is perfectly fine to expect a specific container type here.